env0/resource_environment.go

```diff
@@ -1179,6 +1190,11 @@ func getDeployPayload(d *schema.ResourceData, apiClient client.ApiClientInterfac
 		payload.BlueprintRevision = revision.(string)
 	}
 
+	// For 'Workflows', the 'root' environment should never require a user approval.
+	if _, ok := d.GetOk("sub_environment_configuration"); ok {
```

Comment:
this shouldn't be needed...
This might be a bug in the backend...

Reply:
it seems that the backend will error if this is 'undefined'.

Reply:
this could be overridden by a project policy, this is a good extra step anyways.